### DISCUSSION TITLE
wxwidget could work with webui?

### DISCUSSION THREAD

**Discussion by ifquant** [webui-dev/webui#366] [General]
i need small size program. so i chose wxwidget for gui dev;
 i also need some web ui.   can them could work together?

**Comment by d33p-dev** [comment]
Not sure about WxWidgets and WebUI but I'm about to test the same type of app but with a different native C-based UI library (https://github.com/frang75/nappgui_src).  I've been reading the WebUI source and I'm pretty sure I can make it all work.  I just need to figure out how (in Windows terminology) to make the webUI hWnd be a child window of my native UI parent window's hWnd...

I worked with the maintainer of NApp and he integrated WebView into his library.  But, having the additional features of WebUI might be the direction I need to ultimately take.  Anyway, I'll post here if I have anything useful to add this discussion about integrating webUI within a larger C-based GUI app.

....should work.... but who knows, I'll find out soon and let you know.  If I can make it work with NApp then the same pattern could be applied to WxWidgets.
